tensorflow/lite/kernels/reshape.cc

```diff
@@ -107,8 +107,19 @@ inline TfLiteIntArray* GetOutputShapeFromParam(TfLiteContext* context,
 // Check if the shape tensor is valid. Shapes should be int32 vectors.
 inline bool ShapeIsVector(TfLiteContext* context, TfLiteNode* node) {
   const TfLiteTensor* shape = GetInput(context, node, kShapeTensor);
-  return (shape != nullptr && shape->dims->size == 1 &&
-          shape->type == kTfLiteInt32);
+  bool is_vector;
```

Comment:
Could you add a test case under reshape_test.cc?

Reply:
Also could you add an e2e test case under https://github.com/tensorflow/tensorflow/blob/master/tensorflow/lite/testing/op_tests/reshape.py ?